src/servers/database.ts

```diff
@@ -690,6 +690,7 @@ export class DatabaseServer extends SkeletonServer {
             t.nonce AS "nonce",
             t.value AS "value",
             t.input AS "input",
+            coalesce(t.input, '0x') AS "input",
```

Comment:
@0x3bfc What's the idea here? Now there are two `input` columns in the result set.

Reply:
My bad ... will remove it.